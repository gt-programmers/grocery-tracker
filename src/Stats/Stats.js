import React from 'react';
import './Stats.scss';

const Stats = () => {
  return (
    <section>
      <h3>Welcome to the stats page</h3>
    </section>
  )
}

export default Stats;