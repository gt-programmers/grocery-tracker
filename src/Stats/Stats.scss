.stats {
  display: flex;
  flex-direction: column;
  justify-content: center;
  flex: 1;
  font-size: calc(12px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
}