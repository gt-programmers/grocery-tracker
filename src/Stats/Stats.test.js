import React from 'react';
import Stats from './Stats';
import { shallow } from 'enzyme';

describe("Stats", () => {
  let wrapper = shallow( <Stats />)


  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})


