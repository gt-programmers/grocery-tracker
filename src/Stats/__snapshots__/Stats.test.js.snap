// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stats should match the snapshot 1`] = `
<section
  className="stats"
>
  <h2>
    Welcome, User!
  </h2>
  <h3>
    Update your inventory and see your stats.
  </h3>
</section>
`;
